peekingduck/utils/logger.py

```diff
@@ -83,6 +76,20 @@ def handle_exception(
         self.logger.error(traceback_msg)
         self.logger.error(error_msg)
 
+    @staticmethod
+    def set_log_level(
+        log_level: str = "info", logger_name: Optional[str] = None
+    ) -> None:
+        log_level_settings = set(["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"])
+        log_level = log_level.upper()
+        if log_level not in log_level_settings:
+            log_level = "INFO"
+        logger = logging.getLogger(logger_name)
+        print(f"* logger: name={logger_name}, id={id(logger)}")
```

Comment:
Debugging print statement?

Reply:
Thanks for catching this. And good idea to mention global log level changes (`set_log_level` can make both global and local log level changes). I've added this in the docstrings and also updated the display message to mention global log level changes.